packages/localization/src/dates/getDaysInMonth.js

```diff
@@ -0,0 +1,11 @@
+import CalendarDate from "../../dist/dates/CalendarDate.js";
+
+const getDayInMonth = date => {
+	const tempCalendarDate = new CalendarDate(date);
+	tempCalendarDate.setDate(1);
+	tempCalendarDate.setMonth(tempCalendarDate.getMonth() + 1);
+	tempCalendarDate.setDate(0);
+	return tempCalendarDate.getDate();
+};
+
+export default getDayInMonth;
```

Comment:
the export is called getDayInMonth, the file getDay**s**InMonth (day vs days). The practice is to be called the same

Reply:
Yes, correct for me is days. I changed it. Done